src/common/utils/time/time.ts

```diff
@@ -167,7 +167,16 @@ export function getWallTimeString(date: Date, timezone: Timezone, includeTime?:
   if (includeTime) {
     return wallTimeISOString.replace('T', delimiter || ', ');
   }
-  return wallTimeISOString.replace( /:\d\d/, '').split('T')[0];
+  return splitDateTimeString(wallTimeISOString)[0]
```

Comment:
change date -> format -> custom parse to get date or time 
to
date -> format into date
date -> format into time
directly